diff --git a/node_modules/timeline-arrows/arrow.d.ts b/node_modules/timeline-arrows/arrow.d.ts
index a65a8bd..5695918 100644
--- a/node_modules/timeline-arrows/arrow.d.ts
+++ b/node_modules/timeline-arrows/arrow.d.ts
@@ -185,4 +185,7 @@ export type ArrowOptions = {
      * if true, arrows will be hidden when both items is not visible due to timeline zoom.
      */
     hideWhenItemsNotVisible?: boolean;
+
+    // Custom properties
+    strokeDasharray?: string;
 };
diff --git a/node_modules/timeline-arrows/arrow.js b/node_modules/timeline-arrows/arrow.js
index da79d49..77bd92e 100644
--- a/node_modules/timeline-arrows/arrow.js
+++ b/node_modules/timeline-arrows/arrow.js
@@ -149,6 +149,7 @@ export default class Arrow {
           somePath.style.strokeWidth = this._arrowsStrokeWidth + "px";
           somePath.style.fill = "none";
           somePath.style.pointerEvents = "auto";
+          somePath.style.strokeDasharray = "5, 5"; // Example: 5px dash, 5px space
           this._dependencyPath.push(somePath);
           this._svg.appendChild(somePath);
     }
@@ -274,6 +275,9 @@ export default class Arrow {
             this._dependencyPath[index].setAttribute("d", "M 0 0");
         }
 
+        // Apply strokeDasharray to create dashed effect
+        this._dependencyPath[index].setAttribute("stroke-dasharray", "5, 5"); // Example: 5px dash, 5px space
+
     }
 
     /** @private Función que recibe in Item y devuelve la posición en pantalla del item. */
